app/javascript/components/molecules/StatusDropdown.js

```diff
@@ -18,7 +19,17 @@ export const renderStatusOption = ({ value, label, statusClassName }, { selectVa
   )
 }
 
-const StatusDropdown = ({ status, onChange, disabled, placeholder, size, expand }) => {
+const StatusDropdown = ({
+  status,
+  onChange,
+  overrideValue = false,
+  disabled = false,
+  placeholder = 'Status',
+  size = null,
+  expand = false,
+  minWidthPx = null,
+  updateOnChange = true
```

Comment:
it's best practice to have props default to false, let's rename this to `alwaysDisplayPlaceholderText`